<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UPS Control GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>UPS Discovery</string>
    </property>
    <widget class="QListWidget" name="upsIpList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="itemAlignment">
      <set>Qt::AlignRight</set>
     </property>
    </widget>
    <widget class="QPushButton" name="upsConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Mono</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>621</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabDebug">
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QPushButton" name="dac0DebugSet">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dac0DebugValue">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DAC0 Debug</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dac1DebugValue">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DAC1 Debug</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dac1DebugSet">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pinchValve">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pinch Valve</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledDebugValue">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ledDebugSet">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>LED</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRun">
     <attribute name="title">
      <string>Run</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabLog">
     <attribute name="title">
      <string>Log</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="modeGroupBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QComboBox" name="modeSel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
